doc/man1/flux-uri.rst

```diff
@@ -74,9 +74,16 @@ OPTIONS
 
    Return the *local* (``local://``) equivalent of the resolved URI.
    Warning: the resulting URI may be invalid for the current system
-   if the network host specified by an ``ssh`` URI is not the current
+   if the network host specified by an :program:`ssh` URI is not the current
    host.
```

Comment:
This contains reformatting that isn't mentioned in the commit message.
Might be good to at least call it out.

Reply:
Ok I added that